packages/nx/src/command-line/explain.ts

```diff
@@ -0,0 +1,94 @@
+import * as chalk from 'chalk';
+import { join, relative } from 'path';
+import { nxConfigExplainer } from '../explain/config-explainers/nx-json';
+import { workspaceRoot } from '../utils/app-root';
+import { readJsonFile } from '../utils/fileutils';
+import { output } from '../utils/output';
+
+export interface ExplainArgs {
+  config: string;
+}
+
+export const defaultNxJsonPath = join(workspaceRoot, 'nx.json');
+
+export function explainHandler(args: ExplainArgs) {
+  if (args.config === 'nx.json') {
+    args.config = defaultNxJsonPath;
```

Comment:
This is so that you can be in `apps/app1` and run `--config nx.json`?

I think we should remove this remapping and users should pass `--config ../../nx.json`

Reply:
It also allowed me to set the default value in yargs to be `nx.json` and always have that be resolvable, see `withExplainOptions`.

The user could also provide the relative path you have given, so is there a downside to this?

The alternative would be to remove the default value from yargs and move it into the runtime handler, because we definitely want to allow users to run `nx explain` and have it work without any argument